launcher/src/main/java/org/apache/spark/launcher/CommandBuilderUtils.java

```diff
@@ -328,4 +329,9 @@ static String findJarsDir(String sparkHome, String scalaVersion, boolean failIfN
     return libdir.getAbsolutePath();
   }
 
+  static String getJarsDir(String sparkHome) {
+    String jarsDir = System.getenv(ENV_SPARK_JARS_DIR);
+    return jarsDir != null ? jarsDir : join(File.separator, sparkHome, "jars");
+  }
+
```

Comment:
nit: remove the blank.

Reply:
This is because most of the code in launcher put a blank line after the last method. Shoud I fix it?

Reply:
It's trivial either way. I personally prefer a blank line here as other methods are followed with a blank line.